components/Report.vue

```diff
@@ -190,27 +190,30 @@
         </div>
         <div v-if="!dataPresent" class="pb-3" />
       </section>
-      <section class="section" v-if="climateData">
+      <section class="section large-screen" v-if="climateData">
         <div id="temperature">
           <TempReport />
         </div>
       </section>
-      <section class="section" v-if="climateData">
+      <section class="section large-screen" v-if="climateData">
         <div id="precipitation">
           <PrecipReport />
         </div>
       </section>
-      <section class="section">
+      <section class="section large-screen">
         <div id="permafrost" v-if="permafrostData || type != 'latLng'">
           <PermafrostReport />
         </div>
       </section>
-      <section class="section" v-if="flammabilityData || vegChangeData">
+      <section class="section large-screen" v-if="flammabilityData || vegChangeData">
         <div id="wildfire">
           <WildfireReport />
         </div>
         <BackToTopButton />
       </section>
+      <section class="section !large-screen">
```

Comment:
The `!large-screen` doesn't work here; the "rotate your screen" warning is shown regardless of the size of the viewport.  I think what you want instead is to add a CSS class to handle this, like:

```
@media screen and (min-width: 481px) {
   .large-screen {
     display: block;
   }
 }
```

(Would need to play around with that in practice to see how it works).